beacon_node/beacon_chain/src/beacon_fork_choice_store.rs

```diff
@@ -162,9 +164,9 @@ where
     /// It is assumed that `anchor` is already persisted in `store`.
     pub fn get_forkchoice_store(
         store: Arc<HotColdDB<E, Hot, Cold>>,
-        anchor: &BeaconSnapshot<E>,
+        anchor: BeaconSnapshot<E>,
```

Comment:
Do we need ownership here? we're cloning the beacon state below, and i dont see this moved anywhere